docs/External-Commands.md

```diff
@@ -56,36 +56,38 @@ External commands can be hosted in a [tap](Taps.md) to allow users to easily ins
 
 External commands should be added to a `cmd` directory in the tap. An external command `extcmd` implemented as a Ruby command should live in `cmd/extcmd.rb` (don't forget to `chmod +x`).
 
-To easily use Homebrew's argument parser, replicate the following Ruby template for external commands (replacing all instances of `foo` with the name of the command):
+To easily use Homebrew's argument parser, replicate the Ruby template below for external commands. Your implementation must include the following:
+
+- The class name should be the command name in CamelCase (e.g. `my-cmd` should be named `MyCmd`).
+- Provide a `cmd_args` block that describes the command and its arguments.
+- Implement the `run` method, which will be invoked when the command is executed. Within the `run` method, the parsed arguments are available asusing `args`.
```

Comment:
```suggestion
- Implement the `run` method, which will be invoked when the command is executed. Within the `run` method, the parsed arguments are available using `args`.
```

Nit: Spelling

It might make sense to mention that `args` is available in all instance methods too.